fastserde/avro-fastserde-tests-common/src/test/avro/logicalTypesTest1.avsc

```diff
@@ -0,0 +1,203 @@
+{
+  "type": "record",
+  "name": "FastSerdeLogicalTypesTest1",
+  "namespace": "com.linkedin.avro.fastserde.generated.avro",
+  "doc": "Used in tests to confirm fast-serde supports logical-types",
+  "fields": [
+    {
+      "name": "enumField",
```

Comment:
What do we need this enum type for? Enums are already sufficiently tested elsewhere.

Reply:
There is/was an issue when fast-generic-serializer encodes specific class with enums.
The fix has been extracted to separate PR:
- https://github.com/RTBHOUSE/avro-util/pull/4